LayoutTests/platform/mac-site-isolation/TestExpectations

```diff
@@ -479,7 +479,6 @@ http/tests/security/xss-DENIED-non-shadowable-propterty-with-iframe-proto.html [
 http/tests/security/xss-DENIED-regular-propterty-with-iframe-proto.html [ Skip ]
 http/tests/site-isolation/iframe-process-termination-after-navigation-completed.html [ Skip ]
 http/tests/site-isolation/iframe-process-termination.html [ Skip ]
-http/tests/site-isolation/selection-focus.html [ Skip ]
```

Comment:
I think I've seen other tests hit this stack overflow. Can you retry the crashed tests with this change?